pmm/mysql_queries.go

```diff
@@ -197,6 +197,9 @@ func (a *Admin) RemoveMySQLQueries() error {
 	if err != nil {
 		return err
 	}
+	if data == nil {
+		return fmt.Errorf("can't get key %s", key)
+	}
```

Comment:
I got ugly panic when my malfunction program removed the key and above method was expecting it. Let's handle this with err instead of panic.